crates/joystick/src/windows.rs

```diff
@@ -18,9 +18,6 @@ impl Default for Joystick {
         device.plugin().unwrap();
 
         let gamepad = XGamepad {
-            buttons: vigem_client::XButtons!(
-                UP | DOWN | LEFT | RIGHT | LTHUMB | RTHUMB | LB | RB | GUIDE | A | B | X | Y
```

Comment:
Note: Setting this pressed all the buttons when the joystick is initialized, which is not what we want.

Reply:
yea the xbuttons are set up for you in the driver on create, evdev requires you to set up every button you want to customize your joystick on init (for racing wheels, joysticks with only a couple of buttons etc)